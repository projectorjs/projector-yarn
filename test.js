// @flow
'use strict';

const yarn = require('./');

test('.run()', () => {
  return yarn.run({
    name: 'fixture',
    flags: ['--flag']
  }).then(result => {
    expect(result).toMatchSnapshot();
  });
});
