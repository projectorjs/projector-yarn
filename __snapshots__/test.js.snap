// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`.run() 1`] = `
Object {
  "stderr": "",
  "stdout": "WORKED --flag
",
}
`;
